pkg/analyzer/analyzers/launchdarkly/requests.go

```diff
@@ -0,0 +1,779 @@
+package launchdarkly
+
+import (
+	_ "embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+)
+
+var (
+	baseURL = "https://app.launchdarkly.com/api"
+
+	endpoints = map[string]string{
+		"callerIdentity": "/v2/caller-identity",
```

Comment:
question: How come these three are raw strings instead of constants like the others? 🤔 If it was done on purpose, would you mind just including a brief comment as to why.

Reply:
Yes, this is intentional. These three APIs are specifically for user information, not resources. Additionally, these keys are used only once, unlike resource keys, which are used in multiple places. I'll add a comment to clarify this.